take into account $CFLAGS and $CXXFLAGS set in build environment when building ParaFly
author: Lara Peeters (HPC-UGent)
diff -ru EVidenceModeler.orig/Makefile EVidenceModeler/Makefile
--- EVidenceModeler.orig/Makefile	2024-10-10 09:25:20.000000000 +0200
+++ EVidenceModeler/Makefile	2024-10-16 10:58:57.509308850 +0200
@@ -6,7 +6,7 @@
 CC = gcc
 
 parafly:
-	cd plugins/ParaFly && sh ./configure --prefix=`pwd` CXX=$(CXX) CC=$(CC) CFLAGS="-fopenmp" CXXFLAGS="-fopenmp" && $(MAKE) install 
+	cd plugins/ParaFly && sh ./configure --prefix=`pwd` CXX=$(CXX) CC=$(CC) CFLAGS="$(CFLAGS) -fopenmp" CXXFLAGS="$(CXXFLAGS) -fopenmp" && $(MAKE) install 
 
 
 
